build-s2i.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+VERSION="0.1-rc1"
+MAINTAINERS="Zak Hassan"
+COMPONENT="anomaly-detection-training"
+
+#cleaning up the image folder:
+
+DKR_HUB_NAME=quay.io/zmhassan/anomaly-detection-training:$VERSION
```

Comment:
If you are using `quay.io` why name the variable `DKR_...`? :)

Reply:
👍  Good question. Will replace that with `QUAY...`

Reply:
How about `PUBLIC_IMAGE_NAME` or something like that?

Reply:
Adding that now.

Reply:
Yup fixed this.